src/ui/BackButton/BackButton.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { withDesign } from 'storybook-addon-designs';
+
+import BackButtonComponent from './BackButton';
+
+export default {
+  title: 'Design System/Components/BackButton',
```

Comment:
no longer need the title

Reply:
Removed with https://github.com/coordinape/coordinape/pull/1595/commits/3b733921529d2f7f73f9b810ede7160aaeab6525